devfiles/src/classes/options/NumericInput.tsx

```diff
@@ -104,9 +101,18 @@ export default class NumericInput extends InputOption {
         );
     }
 
-    public callback(): void {
-        this.value = this.newValue;
+    private debounce(newValue: number) {
+        this.bouncyValue = newValue;
+        this.refreshComponent();
+        if (this.timer !== undefined) clearTimeout(this.timer);
+        this.timer = setTimeout(() => {
```

Comment:
Could you please add `"async" () => ...` here and see if you find it more responsive afterward with lots of data?